tests/aliasing/statblock_test.py

```diff
@@ -1,45 +0,0 @@
-from aliasing.api.statblock import AliasSkill
```

Comment:
Why was this file deleted?

Reply:
I renamed the test file from `statblock_test` to `functions_test` since that made more sense.